ats/pkb/pkb.py

```diff
@@ -0,0 +1,57 @@
+from ats.ast import nodes
+
+
+def process_follows(query, statements):
+    import json
+
+    print(json.dumps(query, indent=2))
+
+    return []
+
+
+def create_follows_dictionary(tree: nodes.ProgramNode):
+    parent_childrens_list = []
+    follows_dictionary = {}
+
+    def create_parent_childrens_list(node: nodes.ASTNode):
+        childrens = []
+        for i in range(len(node.children)):
+            create_parent_childrens_list(node.children[i])
+            childrens.append(node.children[i])
+        if len(node.children) > 0:
+            parent_childrens_list.insert(0, childrens)
```

Comment:
Czemu akurat tak? Nie rozumiem. Ten kawalek kodu wykonuje sie jak konczycie obrobke konkretnego node i chcecie wyjsc wyzej. Mieszaja wam sie totalnie indeksy teraz. Zalozmy taki przypadek:
```
    stmt   
   /    \
stmt    stmt
```

Poprawnie ponumerowany powinien wygladac tak:

```
     stmt1
    /     \
stmt2     stmt3
```
Tak sie sklada, ze to jest dokladnie taka sama numeracja jak kolejnosc z jaka wchodzicie do nodeow uzywajac DFSa.

Natomiast ta linijka sprawia, ze numerujecie je tak:

```
     stmt1
    /     \
stmt3     stmt2
```